README.md

```diff
@@ -98,6 +98,15 @@ Hurray! You successfully made a contribution!
 
 ---
 
+## <ins> What if I have a Conflict? </ins>
+
+#### A GitHub conflict is when people make changes to the same area or line in a file. This must be fixed before it is merged in order to prevent collision in the main branch. 
+
+* #### To read more about this, go to [Github Docs - About Merge Conflicts](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/about-merge-conflicts)
+* #### To find out about how to fix a Git Conflict, go to [Github Docs - Resolve Merge Conflict](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)
```

Comment:
```suggestion
* #### To find out about how to fix a Git Conflict, go to the [Github Docs - Resolve Merge Conflict](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) page
```

Reply:
Same point as I mentioned in the above comment.